### ISSUE TITLE
body get hidden when import model to A2+ WB

### ISSUE THREAD

**Issue by QuakeString** [realthunder/FreeCAD#133]
Cant see some part of body in A2+ work bench. I'm not sure if I'm trying something wrong. or its issue of A2+ Workbench and may be I need to report to A2+ Workbench github. Your advice will be appriciated. [Here is link of FreeCAD model file.](https://www.dropbox.com/sh/443isukcqjyixh9/AADvBI94SMiH7LX1imSGIHkla?dl=0)

![Screenshot_20210430_233501](https://user-images.githubusercontent.com/5375918/116736540-a2f20980-aa0d-11eb-8ef4-27a5a9c03095.png)
![Screenshot_20210430_233423](https://user-images.githubusercontent.com/5375918/116736550-a7b6bd80-aa0d-11eb-9803-245f593a5fd1.png)

**Comment by realthunder** [comment]
I have submit a [PR](https://github.com/kbwbe/A2plus/pull/428) to A2plus for the fix.

**Comment by kbwbe** [comment]
The PR has been merged to A2plus...

**Comment by QuakeString** [comment]
Thank You !

**Comment by QuakeString** [comment]
Could it be the issue for the same A2+ WB or something else ?
FreeCAD Using too much memory and its also performing poorly. 
![Screenshot_20210509_140623](https://user-images.githubusercontent.com/5375918/117565638-df89c900-b0cf-11eb-9538-68b5c060fc8a.png)
I've opened the same model with FreeCAD 0.19.1, using less then half memory and its also very fast to use, there is no lag in zooming.
![Screenshot_20210509_145031](https://user-images.githubusercontent.com/5375918/117566775-4ad69980-b0d6-11eb-83f4-d5592aa1f02e.png)

**Comment by realthunder** [comment]
Can you please paste the FreeCAD about information here? Are you using my Stable release? If so, can you please try Daily?

**Comment by QuakeString** [comment]
OS: Manjaro Linux (KDE/plasma)
Word size of OS: 64-bit
Word size of FreeCAD: 64-bit
Here is FreeCAD Link 
![Screenshot_20210509_150329](https://user-images.githubusercontent.com/5375918/117567045-bd944480-b0d7-11eb-96bf-2264f69814b2.png)

Version: 2021.504.24301 +3511 (Git) AppImage
Build type: Release
Branch: LinkStage3
Hash: b8cd87c34a66a8f7a8177f483cebf79e29273813
Python version: 3.8.8
Qt version: 5.12.9
Coin version: 4.0.1
OCC version: 7.4.0
Locale: English/United States (en_US)

**Comment by QuakeString** [comment]
I'll Try Daily.
OS: Manjaro Linux (KDE/plasma)
Word size of OS: 64-bit
Word size of FreeCAD: 64-bit
Version: 2021.504.24301 +3511 (Git) AppImage
Build type: Release
Branch: LinkStage3
Hash: b8cd87c34a66a8f7a8177f483cebf79e29273813
Python version: 3.8.8
Qt version: 5.12.9
Coin version: 4.0.1
OCC version: 7.4.0
Locale: English/United States (en_US)
![Screenshot_20210509_150329](https://user-images.githubusercontent.com/5375918/117567062-d1d84180-b0d7-11eb-8534-1fce1271fe21.png)

**Comment by QuakeString** [comment]
Result is quite similar.  
** Mistakenly I've "Closed" the issue.**

I've noticed one more thing, if I open my assembly and try to add my model, that time it take too long time and too much RAM, but if that model already in assembly and I open that assembly it uses 50%  of RAM.
![image](https://user-images.githubusercontent.com/5375918/117567677-be7aa580-b0da-11eb-88a4-6932f43e784f.png)
![Screenshot_20210509_152246](https://user-images.githubusercontent.com/5375918/117567706-e79b3600-b0da-11eb-941e-9cced179cdcc.png)

**Comment by realthunder** [comment]
You are testing using the same model as the OP right? The extra memory could be held in undo stack. My branch does use extra memory for topo naming. I'll investigate more.

**Comment by QuakeString** [comment]
Yes, I've used exact same model. the same model I had problem with A2+ WB, it is now sorted after A2+ WB update.

**Comment by realthunder** [comment]
What are the exact steps you did to experience the memory problem. I loaded up your `assembly_02` file, and the memory usage is 1.3GB. Then I click A2Plus edit assembly button to load the the `rotary_dryer_main_body_MOD02.FCStd`, it increases to 1.6GB. Then I click to reveal the hidden `DrumBody001`, it goes to 1.8GB.

Can you send me your `link.user.cfg` file located in directory `~/.FreeCAD`.

**Comment by QuakeString** [comment]
here is link of files
https://drive.google.com/file/d/14frRh4dZwX5jazburxnE6j0P-lm0KLzT/view?usp=sharing
https://drive.google.com/file/d/1escskqqwatV1YTLm-kG8GUd_gAGBZ38g/view?usp=sharing

**Comment by luzpaz** [comment]
bump

**Comment by luzpaz** [comment]
@QuakeString can you zip the `link.user.cfg` file and attach it to this ticket? The google drive URLs are invalid now
